source/install/cloud-changelog.md

```diff
@@ -3,12 +3,56 @@
 This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/get-started/), an enterprise-grade SaaS offering hosted by Mattermost.
 
 Latest Mattermost Cloud releases:
+- [Release 2021-10-13](#release-2021-10-13)
 - [Release 2021-09-29](#release-2021-09-29)
 - [Release 2021-08-12](#release-2021-08-12)
 - [Release 2021-07-29](#release-2021-07-29)
 - [Release 2021-07-15](#release-2021-07-15)
 - [Release 2021-07-01](#release-2021-07-01)
-- [Release 2021-06-16](#release-2021-06-16)
+
+## Release 2021-10-13
+
+### Improvements
+
+#### User Interface (UI)
+ - Added a query param to translate in-product help pages when opened from the Desktop App.
+ - Added rendering for posts containing markdown in email notifications.
+ - Added support for inline Latex rendering.
+ - Added the **Move to...** option menu item to the channel header dropdown.
+ - Added keyboard shortcuts to tooltips. Use shortcut key component for displaying keys.
+ - Changed the user interface of the edit-indicator of posts and moved it inline.
+ - Added support for Global threads infinite scroll.
+
+#### Integrations
+ - Added support for multi-select on Apps slash commands.
+ - App commands now make a distinction between the central channel and the right-hand side channel.
+ - App bindings now recognize channels in between in the post menus.
+ - Added new ``registerMessageWillBeUpdatedHook(newPost, oldPost)`` client-side plugin hook to intercept edited messages.
+
+#### Performance
+ - Added performance improvements for draft storage with multiple tabs open.
+ - Improved performance of draft loading.
+ - Slightly improved performance around rendering of system messages.
+
+#### Administration
+ - Bulk imports with attached files now log and continue when a file fails to upload instead of halting.
+ - The Bleve index has been updated to v2 to use the scorch index type. This new default index type has some improvements on efficiency, which means that the indexes use significantly less disk space. To use this new type of index, after upgrading the server version, run a purge operation and then a reindex from the Bleve section of the System Console. Bleve is still compatible with the old indexes, so the currently indexed data will work fine if the purge and reindex is not run.
+ - ``get flagged posts`` endpoint will now return only flagged posts for channels the user is member of.
+ - Minimum supported browser versions changes:
```

Comment:
Do these browser minimum version updates apply to both Cloud and self-hosted deployments?

We have different values documented in the product docs: https://docs.mattermost.com/install/software-hardware-requirements.html#software-requirements -- do we want to update these docs as well?

Reply:
@esethna or @jasonblais Can one of you can confirm?

Reply:
@amyblais are these changes following the software requirements update process, or anything we're doing outside of that?

Reply:
Thanks! I'll ensure the minimum browser and OS requirements noted below in this change log are reflected in the product docs.